base/recipes/gearman.rb

```diff
@@ -88,6 +90,7 @@
 # Memcached seems to be sideloaded by gearrman; disable it
 service "memcached" do
   supports status: true, restart: true, reload: true, stop: true
+  provider Chef::Provider::Service::Systemd
```

Comment:
This is clutch, thanks for finding/including this!

Reply:
that doesn't actually do anything btw, just included for consistency